udata_front/theme/gouvfr/templates/user/base.html

```diff
@@ -24,102 +25,97 @@
 {% endblock %}
 
 {% block toolbar %}
+{% if current_user != user %}
+    {{ follow_btn(user) }}
+{% endif %}
 {% if can_edit %}
-<div class="btn-group btn-group-xs">
-    <a href="{{ url_for('admin.index', path='users/{id}/'.format(id=user.id)) }}" class="btn btn-success">
-        <span class="fa fa-pencil"></span>
+    {% if current_user != user %}
+    <a
+        href="{{ url_for('admin.index', path='user/{id}/'.format(id=user.id)) }}"
+        class="fr-ml-2w fr-btn fr-btn--sm fr-btn--secondary fr-btn--secondary-warning-425 fr-icon-pencil-line fr-btn--icon-left"
+    >
+        {{ _("User's admin profile") }}
+    </a>
+    {% else %}
+    <a
+        class="fr-ml-2w fr-btn fr-btn--sm fr-btn--secondary fr-btn--secondary-warning-425 fr-icon-pencil-line fr-btn--icon-left"
+        href="{{ url_for('admin.index', path='me') }}"
+    >
         {{ _('Edit') }}
     </a>
-</div>
-{% endif %}
-{# Social actions #}
-{% if current_user != user %}
-<div class="btn-group btn-group-xs">
-    {{ follow_btn(user) }}
-</div>
+    {% endif %}
 {% endif %}
 {% endblock %}
 
 {% block main_content %}
-<div class="fr-container">
-    <div class="fr-alert fr-alert--info">
-        <p class="fr-alert__title">{{ _('Work in progress') }}</p>
-        <p>{{ _('The user page is currently being renovated to improve user experience.') }}</p>
-    </div>
-    <div class="fr-my-2w fr-grid-row fr-grid-row--gutters">
-        <div class="fr-col-md-3">
-            <div class="panel panel-default">
-                <div class="panel-body">
-                    <div class="text-center">
-                        <img src="{{ user|avatar_url(150) }}"
-                            alt="{{ user.fullname }}" class="scalable"/>
-                    </div>
-                    <h2>{{ user.fullname }}</h2>
-                    {{ user.about|markdown }}
-                    {# <hr/> #}
-                    <div>
-                        <ul class="list-unstyled">
-                            {% if user.website %}
-                            <li>
-                                <span class="fa fa-link"></span>
-                                <a href="{{user.website}}" rel="nofollow">{{user.website}}</a>
-                            </li>
-                            {% endif %}
-                        </ul>
-                    </div>
-                </div>
-
-                <div class="list-group text-center">
-                    <a href="{{url_for('users.show', user=user)}}"
-                        class="list-group-item {% if user_tab == 'activity' %}active{% endif %}">
-                        <strong>{{ _('Activity') }}</strong>
-                    </a>
-                    <a href="{{url_for('users.followers', user=user)}}"
-                        class="list-group-item {% if user_tab == 'followers' %}active{% endif %}">
-                        <strong>
```

Comment:
The `followers` and `followings` views still exist, but we don't have any direct link to these. Should we keep a link to these on the user page? Or remove it entirely? cc @agarrone

Reply:
Given that they are not accessibles through the administration it would be nice to have dedicated pages for those but only accessible to the authenticated user. 
Something like this would do the job 
![image](https://github.com/etalab/udata-front/assets/60264344/8d241d69-964d-4762-8547-3005a9d17149)

Reply:
authenticated & admin user please :p